---
# layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

 A list is also available at [google scholar](https://scholar.google.com/citations?user=FWmDPGgAAAAJ&hl=en) or [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/?term=Bosdriesz+E). 
 
\# indicates equal contribution. Mentees’ names in bold.

## Key

N. Kramer, R. van Eijl, **T. Stohn**, S. Tanis, L.F.A. Wessels, **E. Bosdriesz<sup>#</sup>**, and K.W. Mulder<sup>#</sup>. 2025. “Cell-State Specific Drug-Responses Are Associated with Differences in Signaling Network Wiring.” bioRxiv. doi:[10.1101/2025.01.27.635060](https://doi.org/10.1101/2025.01.27.635060), 2025.<br>
*Shows that both the the response of signaling proteins and the interactions between proteins depend on cell-state*
<a href="https://github.com/evertbosdriesz/scIDseq-CNR"> Notebooks </a>

**T. Stohn**, R. van Eijl, K.W. Mulder, L.F.A. Wessels, and **E. Bosdriesz**, “Reconstructing and Comparing Signal Transduction Networks from Single Cell Protein Quantification Data.”  *Bioinformatics* doi:[10.1093/bioinformatics/btaf675](https://doi.org/10.1093/bioinformatics/btaf675), 2025.<br>
*Method to reconstruct and compare signaling networks subpopulation specific signaling networks from single cell (phospho)-protein quantification data.*<br>
<i class="fab fa-github"></i> <a href="https://github.com/ibivu/scmra"> Code </a> -
<a href="https://github.com/tstohn/scmra_analysis"> Notebooks </a>

**E. Bosdriesz**, J. Fernandes Neto, A. Sieber, R. Bernards,N. Bl&uuml;thgen and L.F.A. Wessels, 'Identifying mutant-specific drug combinations using Comparative Network Reconstruction'. *iScience*, *25*(8); doi:[10.1016/j.isci.2022.104760](https://dx.doi.org/10.1016/j.isci.2022.104760), 2022.<br>
*Shows that Comperative Network Reconstruction can be be used to predict which low-dose multi-drug combinations that are likely to selective, using an isogenic cell-line pair.*<br>
<i class="fab fa-github"></i> <a href="https://github.com/evertbosdriesz/cnr-selective-combos"> Code and Data </a>

J. Fernandes Neto, E. Nadal<sup>#</sup>, **E. Bosdriesz**<sup>#</sup>, S. N. Ooft, L. Farre, C. McLean, S. Klarenbeek, A. Jurgens, H. Hagen, L. Wang, E. Felip, A. Martinez-Marti, A. Vidal, E. Voest, L.F.A. Wessels, O. van Tellingen, A. Villanueva and & R. Bernards, 'Multiple Low Dose Therapy as an Effective Strategy to Treat EGFR Inhibitor-Resistant NSCLC Tumours.' *Nature Communications*, *11 (1)*: 3157. doi:[10.1038/s41467-020-16952-9](https://doi.org/10.1038/s41467-020-16952-9), 2020.<br>
*Shows that combining multiple drugs at low dose is effective, prevents emergence resistance, and is well tolerated by mice.*

**E. Bosdriesz**, A. Prahallad, B. Klinger, A. Sieber, A. Bosma, R. Bernards, N. Bl&uuml;thgen and L.F.A. Wessels, ‘Comparative Network Reconstruction Using Mixed Integer Programming’, *Bioinformatics*, *34*:i997–1004. doi:[10.1093/bioinformatics/bty616](https://doi.org/10.1093/bioinformatics/bty616), 2018.<br>
*Method to reconstruct and compare signaling networks based on perturbation data using Mixed Integer Quadratic programming.*<br>
<i class="fab fa-github"></i> <a href="https://github.com/NKI-CCB/cnr"> Code </a> -
<a href="https://github.com/NKI-CCB/CNR-analyses"> Notebooks </a>
<!-- 
A. Ressa<sup>#</sup>, **E. Bosdriesz**<sup>#</sup>, J. de Ligt, S. Mainardi, G. Maddalo, A. Prahallad, M. Jager, L. de la Fonteijne, M. Fitzpatrick, S. Groten, A.F.M. Altelaar, R. Bernards, E. Cuppen, L.F.A Wessels and J.R. Heck,
‘A System-Wide Approach to Monitor Responses to Synergistic BRAF and EGFR Inhibition in Colorectal Cancer Cells’,
*Molecular & Cellular Proteomics*, *17*(10):1892-1908.
doi:[10.1074/mcp.RA117.000486](https://doi.org/10.1074/mcp.RA117.000486), 2018.<br>
*Transcriptomic, proteomic and phosphoproteomic, time-course analysis of cancer cells in response to targeted inhibitors*:<br>
<a href="https://bitbucket.org/evertbosdriesz/cgc-multi-omics/src/master/"><i class="fab fa-bitbucket"></i> Code </a> -->

**E. Bosdriesz**, D. Molenaar, B. Teusink, and F.J. Bruggeman,
‘How Fast-Growing Bacteria Robustly Tune Their Ribosome Concentration to Approximate Growth-Rate Maximization.’,
*FEBS Journal*, *282*(10):2029-2044. doi:[10.1111/febs.13258](https://doi.org/10.1111/febs.13258), 2015.<br>
*Uses mechanistic, minimal models to understand the design principles of how bacteria control their macromolecular composition*:<br>
<a href="https://jjj.bio.vu.nl/models/bosdriesz1/"><i class="fas fa-home"></i> model</a>

## Preprints

N. Kramer, R. van Eijl, **T. Stohn**, S. Tanis, L.F.A. Wessels, **E. Bosdriesz<sup>#</sup>**, and K.W. Mulder<sup>#</sup>. 2025. “Cell-State Specific Drug-Responses Are Associated with Differences in Signaling Network Wiring.” bioRxiv. https://doi.org/10.1101/2025.01.27.635060.
<a href="https://github.com/evertbosdriesz/scIDseq-CNR"> Notebooks </a>


## Peer-reviewed articles

**T. Stohn**, R. van Eijl, K.W. Mulder, L.F.A. Wessels, and **E. Bosdriesz**, “Reconstructing and Comparing Signal Transduction Networks from Single Cell Protein Quantification Data.”  *Bioinformatics* doi:[10.1093/bioinformatics/btaf675](https://doi.org/10.1093/bioinformatics/btaf675), 2025.<br>

**E. Bosdriesz**, J. Fernandes Neto, A. Sieber, R. Bernards,N. Bl&uuml;thgen and L.F.A. Wessels, 'Identifying mutant-specific drug combinations using Comparative Network Reconstruction'. *iScience*, *25*(8); doi:[10.1016/j.isci.2022.104760](https://dx.doi.org/10.1016/j.isci.2022.104760), 2022.


J. Fernandes Neto, E. Nadal<sup>#</sup>, **E. Bosdriesz**<sup>#</sup>, S. N. Ooft, L. Farre, C. McLean, S. Klarenbeek, A. Jurgens, H. Hagen, L. Wang, E. Felip, A. Martinez-Marti, A. Vidal, E. Voest, L.F.A. Wessels, O. van Tellingen, A. Villanueva and & R. Bernards, 'Multiple Low Dose Therapy as an Effective Strategy to Treat EGFR Inhibitor-Resistant NSCLC Tumours.' *Nature Communications*, *11* (1): 3157.doi:[10.1038/s41467-020-16952-9](https://doi.org/10.1038/s41467-020-16952-9), 2018.

T. Sustic, **E. Bosdriesz**, S. van Wageningen, L.F.A. Wessels and R. Bernards, 'RUNX2/CBFB modulates the response to MEK inhibitors through activation of receptor tyrosine kinases in *KRAS*-mutant colorectal cancer', *Translational Oncology*, *13*(2):201-211.
doi:[10.1016/j.tranon.2019.10.006](https://doi.org/10.1016/j.tranon.2019.10.006), 2020.

**E. Bosdriesz**, A. Prahallad, B. Klinger, A. Sieber, A. Bosma, R. Bernards, N. Blüthgen and L.F.A. Wessels, ‘Comparative Network Reconstruction Using Mixed Integer Programming’, *Bioinformatics*, *34*:i997–1004. doi:[10.1093/bioinformatics/bty616](https://doi.org/10.1093/bioinformatics/bty616), 2018.

A. Ressa<sup>#</sup>, **E. Bosdriesz**<sup>#</sup>, J. de Ligt, S. Mainardi, G. Maddalo, A. Prahallad, M. Jager, L. de la Fonteijne, M. Fitzpatrick, S. Groten, A.F.M. Altelaar, R. Bernards, E. Cuppen, L.F.A Wessels and J.R. Heck ,
‘A System-Wide Approach to Monitor Responses to Synergistic BRAF and EGFR Inhibition in Colorectal Cancer Cells’,
*Molecular & Cellular Proteomics*, *17*(10):1892-1908.
doi:[10.1074/mcp.RA117.000486](https://doi.org/10.1074/mcp.RA117.000486), 2018.

T. Šuštić<sup>#</sup>, S. van Wageningen<sup>#</sup>, **E. Bosdriesz**, R.J.D. Reid, J. Dittmar, C. Lieftink,  R.L. Beijersbergen, L.F.A. Wessels, R. Rothstein and R. Bernards, ‘A Role for the Unfolded Protein Response Stress Sensor ERN1 in Regulating the Response to MEK Inhibitors in KRAS Mutant Colon Cancers’,
*Genome Medicine*, *10*:90. doi:[10.1186/s13073-018-0600-z](https://doi.org/10.1186/s13073-018-0600-z), 2018.

L. Wang#, R. Leite de Oliveira#, S. Huijberts, **E. Bosdriesz**, N. Pencheva, D. Brunen, A. Bosma, J.Y. Song, J. Zevenhoven, G. T. Los-de Vries, H. Horlings, B. Nuijen, J.H. Beijnen, J.H.M. Schellens and R. Bernards.
‘An Acquired Vulnerability of Drug-Resistant Melanoma with Therapeutic Potential’.
*Cell*, *173*(6):1413-1425.e14. doi:[10.1016/j.cell.2018.04.012](https://doi.org/10.1016/j.cell.2018.04.012), 2018.

M. Dorel, B. Klinger, T. Gross, A. Sieber, A. Prahallad, **E. Bosdriesz**, L.F.A. Wessels and N. Blüthgen, ‘Modelling Signalling Networks from Perturbation Data’, *Bioinformatics*, *34*(23):4079–4086. doi:[10.1093/bioinformatics/bty473](https://doi.org/10.1093/bioinformatics/bty473), 2018.

Z. Xue, D.J. Vis, A. Bruna, T. Šuštić, S. Van Wageningen, A. Sati Batra, O.M. Rueda, **E. Bosdriesz**, C. Caldas, L.F.A. Wessels and R. Bernards. ‘MAP3K1 and MAP2K4 Mutations Are Associated with Sensitivity to MEK Inhibitors in Multiple Cancer Models’.
*Cell Research*, *28*:719–729. doi:[10.1038/s41422-018-0044-4](https://doi.org/10.1038/s41422-018-0044-4), 2018.

**E. Bosdriesz**<sup>#</sup>, M.T. Wortel<sup>#</sup>, J.R. Haanstra, M.J. Wagner, P. de la Torre Cortés, and B. Teusink, ‘Low Affinity Uniporter Carrier Proteins Can Increase Net Substrate Uptake Rate by Reducing Efflux’, *Scientific Reports*, *8*:5576. doi:[10.1038/s41598-018-23528-7](https://doi.org/10.1038/s41598-018-23528-7), 2018.

M.T. Wortel<sup>#</sup>, **E. Bosdriesz**<sup>#</sup>, B. Teusink, and F.J. Bruggeman.
‘Evolutionary Pressures on Microbial Metabolic Strategies in the Chemostat’.
*Scientific Reports*, *6*:29503. doi:[10.1038/srep29503](https://doi.org/10.1038/srep29503), 2016.

**E. Bosdriesz**, D. Molenaar, B. Teusink, and F.J. Bruggeman, ‘How Fast-Growing Bacteria Robustly Tune Their Ribosome Concentration to Approximate Growth-Rate Maximization.’, *FEBS Journal*, *282*(10):2029-2044. doi:[10.1111/febs.13258](https://doi.org/10.1111/febs.13258), 2015.

**E. Bosdriesz**, S. Magnúsdóttir, F.J. Bruggeman, B. Teusink, and D. Molenaar,
‘Binding Proteins Enhance Specific Uptake Rate by Increasing the Substrate-Transporter Encounter-Rate’,
*FEBS Journal*, *282*:2394–2407. doi:[10.1111/febs.13289](https://doi.org/10.1111/febs.13289), 2015.

J. Berkhout, **E. Bosdriesz**, E. Nikerel, D. Molenaar, D. de Ridder, B. Teusink and F.J. Bruggeman,
‘How Biochemical Constraints of Cellular Growth Shape Evolutionary Adaptations in Metabolism.’,
*Genetics*, *194*:505–512. doi:[10.1534/genetics.113.150631](https://doi.org/10.1534/genetics.113.150631), 2013.

<br>

<!-- 
Mentees’ names underlined. Asterisk indicates dual first-author position.
**Articles provided for personal use only.**

You can also find my articles on <u><a href="https://scholar.google.com/citations?user=YRWfuEIAAAAJ&hl=en&oi=sra">my Google Scholar profile</a>.</u>

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %} -->
